docs/install.md

```diff
@@ -0,0 +1,58 @@
+<!--
+SPDX-FileCopyrightText: 2021 Markus Sommer
+
+SPDX-License-Identifier: GPL-3.0-or-later
+-->
+
+# Installation Instructions
+
+## Manual
+
+- Install the *Go Programming language*, either through your package manager, or from [here](https://golang.org/dl/).
+- Clone the [source repository](https://github.com/dtn7/dtn7-go).
+- Check out the most recent release tag. Or don't and just build the `HEAD`, but we don't promise that it won't be broken.
+- (Optional) Run `GO111MODULE=on go test ./...` in the repository root to make sur we didn't screw up too badly.
+- Run `GO111MODULE=on go build ./...` in the repositry root to build both `dtnd` and `dtn-tool`
+
+If you want to run `dtnd` as a service, you might also want to do the following:
```

Comment:
The following section is very targeted to a specific setup; a generic GNU/Linux with systemd. Thus, this might be moved to the "Other Linux" section. You might wanna also add a reference from, e.g., Arch Linux to this section.